lib/http.js

```diff
@@ -304,6 +308,51 @@ bedrock.events.on('bedrock-express.configure.routes', app => {
     })
   );
 
+  // GET /ledger-agent/{AGENT_ID}/peers
+  // Get recommended peers
+  app.options(routes.peers, cors());
+  app.get(
+    routes.peers, optionallyAuthenticated, cors(),
+    asyncHandler(async (req, res) => {
+      const {actor} = req.user || {};
+      const agentId = 'urn:uuid:' + req.params.agentId;
+      const options = {};
+
+      if(!req.user) {
+        options.public = true;
+      }
+
+      const ledgerAgent = await brLedgerAgentGet(
+        (req.user) ? actor : null, agentId, options);
+      const {ledgerNode} = ledgerAgent;
+
+      const result = await ledgerNode.peers.getRecommended();
```

Comment:
Perhaps leave a comment here that says that all recommended peers will be returned and limiting the number of recommended peers is up to the consensus plugin that marks peers as recommended.